### PR TITLE
TRON-39 - tron wallet recoveries

### PR DESCRIPTION
**Pull request by twtaylorbitgo**
Needs an updated BItGoJS with recover function.

### CODE REVIEW

**Comment by dannybitgo** [comment]
Yeah lets get the new bitgojs in before this lands

File: src/constants/coin-config.js [hunk 1]
```diff
@@ -108,12 +109,21 @@ export default {
         { label: 'Mainnet', value: 'prod' },
         { label: 'Testnet (Kovan)', value: 'test' }
       ]
-    }
+    },
+    trx: {
+      fullName: 'Tron',
+      supportedRecoveries: [],
+      icon: xlmIcon,
```
Comment (argjv) [inline src/constants/coin-config.js]: Icon is wrong

Comment (argjv) [inline src/constants/coin-config.js]: Icon is wrong

Comment (twtaylorbitgo) [reply src/constants/coin-config.js]: should be fixed

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "autoprefixer": "8.6.5",
-    "bitgo": "6.2.0",
+    "bitgo": "8.5.1",
```
[the remark this answers is not in the document]
Comment (twtaylorbitgo) [reply package.json]: 9.0.1 now

File: src/constants/coin-config.js [hunk 3]
```diff
@@ -108,12 +109,21 @@ export default {
...
       ]
-    }
+    },
+    trx: {
+      fullName: 'Tron',
+      supportedRecoveries: [],
+      icon: trxIcon,
+      envOptions: [
+        { label: 'Mainnet', value: 'prod' },
+        { label: 'Testnet', value: 'test' }
```
Comment (argjv) [inline src/constants/coin-config.js]: Should this be `Testnet (Tronex)` like in ethereum?

Comment (twtaylorbitgo) [reply src/constants/coin-config.js]: oh yeah, good idea. ok

**Pull request merged**